tests/unit/amo/components/TestAddon.js

```diff
@@ -44,7 +44,10 @@ import {
   dispatchSignInActions, fakeAddon, signedInApiState,
 } from 'tests/unit/amo/helpers';
 import {
-  createFetchAddonResult, getFakeI18nInst, sampleUserAgentParsed,
+  createFetchAddonResult,
+  getFakeI18nInst,
+  sampleUserAgentParsed,
+  createStubErrorHandler,
```

Comment:
We can use http://eslint.org/docs/rules/sort-imports - I'll file an issue on eslint-config-amo to investigate.